apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -253,6 +262,23 @@ type ExtensionFunctions = {
   functions: AbiFunction[];
 };
 
+const FunctionInputSearch = ({
+  searchFn,
+}: {
+  searchFn: DebouncedState<(e: ChangeEvent<HTMLInputElement>) => void>;
+}) => (
+  <div className="sticky top-0 z-10 mb-3">
+    <div className="relative w-full">
+      <SearchIcon className="-translate-y-1/2 absolute top-[50%] left-3 size-4 text-muted-foreground" />
+      <Input
+        placeholder="Search"
+        className="rounded-none border-r-none border-l-none py-2 pl-9 focus-visible:ring-0"
+        onChange={searchFn}
```

Comment:
why we are not setting the value here?

Reply:
Not sure what you mean here (but I updated the code anyway)

Reply:
`Input` is given `onChange` but not `value` prop

Reply:
pushed the fix